packages/react/src/provider.tsx

```diff
@@ -2,15 +2,23 @@ import * as React from 'react';
 import { Client, OpenFeature } from '@openfeature/web-sdk';
 
 type ProviderProps = {
+  /**
+   * OpenFeature client to use.
+   */
   client?: Client;
+  /**
+   * The name of the client.
+   * @see OpenFeature.setProvider() and overloads.
+   */
+  clientName?: string,
```

Comment:
These are mutually exclusive. Perhaps we should make a union type.

Reply:
That's a good point.

Reply:
Done